graph/bolt/quadstore.go

```diff
@@ -184,7 +185,7 @@ var (
 	metaBucket = []byte("meta")
 )
 
-func (qs *QuadStore) ApplyDeltas(deltas []graph.Delta) error {
+func (qs *QuadStore) ApplyDeltas(deltas []graph.Delta, ignoreDup, ignoreMiss bool) error {
```

Comment:
I'll take it. I might have done a struct for this case, but (at the moment) it's two bools, and may get updated. So, this may get upgraded to full options or to a struct if we add another flag, but for now, I'm willing to go with it.